.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build the application
+        run: |
+          echo "Placeholder for actual build commands"
+          # Replace the above line with actual build commands (like npm run build, etc.)
+
+  # Permissions to request and use OIDC tokens and access repository contents
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
+
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::393416225559:role/GitHubActionsRole
```

Comment:
How does this work? Did we have to do some prior setup to make it work? Is it this? https://github.com/aws-actions/configure-aws-credentials#overview

Reply:
Yes, so I went away and created a custom IAM policy, and attached it to a role.
Have a look at IAM->Roles->GitHubActionsRole. If you expand the associated policy, you'll see more granular permissions. The improvements task that's coming up is to slim this down a bit.

Reply:
Is it worth documenting this in case we have to re-do it in a new AWS subscription? Or did you just follow the README in which case maybe we just need a pointer to that somewhere in one of our docs